nixos/modules/services/networking/v2raya.nix

```diff
@@ -16,7 +16,12 @@ with lib;
       unitConfig = {
         Description = "v2rayA service";
         Documentation = "https://github.com/v2rayA/v2rayA/wiki";
-        After = [ "network.target" "nss-lookup.target" "iptables.service" "ip6tables.service" ];
+        After = [
+          "network.target"
+          "nss-lookup.target"
+          "iptables.service"
+          "nftables.service"
```

Comment:
Why dose v2raya start after both nftables and iptables? Usually, only one of them is used, and v2raya will never be started if both of them is in after. It is better to use `config.networking.nftables.enable` or something like it to determine which one is used.